pkg/linode-bs/luks.go

```diff
@@ -0,0 +1,405 @@
+/*
+Copyright 2019 linkyard ag
+Copyright cloudscale.ch
+Copyright 2022 Akamai Technologies
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package linodebs
+
+import (
```

Comment:
this whole file is a mess and i know you got it that way but let's run goimports across is at clean it up a bit